### PR TITLE
Update intro.html

### PR DESCRIPTION
**Pull request by rbg-joe**
Changed wording and added hyperlink

### CODE REVIEW

File: templates/web/greenwich/around/intro.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <h1>[% loc('Report a problem in Royal Greenwich streets or parks') %]</h1>
 <p>
-    Use this for reporting pysical problems that can be fixed like graffiti,
-    street cleaning issues or road defects. To report other kinds of problems
-    such as missed bins use our <a href="#">reporting forms</a>.
+    Use this for reporting pysical problems that can be fixed like graffiti,street cleaning issues or road defects.
```
Comment (stevenday) [inline templates/web/greenwich/around/intro.html]: @rbg-joe thanks for this! There's a couple of minor typos in this line: 
> Use this for reporting **pysical** problems that can be fixed like **graffiti,street** cleaning issues or road defects.

If you can fix those (happy for you to force push the fixed commit) I'll get this merged in!

File: templates/web/greenwich/around/intro.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <h1>[% loc('Report a problem in Royal Greenwich streets or parks') %]</h1>
 <p>
-    Use this for reporting pysical problems that can be fixed like graffiti,
-    street cleaning issues or road defects. To report other kinds of problems
-    such as missed bins use our <a href="#">reporting forms</a>.
+    Use this for reporting pysical problems that can be fixed like graffiti,street cleaning issues or road defects. 
+</p>
+<p>
+    To report other kinds of problems such as missed bins use our <a href="#">reporting forms</a>.
```
Comment (stevenday) [inline templates/web/greenwich/around/intro.html]: Whilst you're editing this copy, can you confirm what url we should use for this link? I found: https://www.royalgreenwich.gov.uk/site/xfp/scripts/xforms_form.php?formID=45&language=en, but is there a more generic page with other forms on it that we should use instead?

**Comment by rbg-joe** [comment]
Hi
I started again with: 
https://github.com/mysociety/fixmystreet/pull/1122

**Comment by rbg-joe** [comment]
Sorry close that - the link was an old one.

**Comment by stevenday** [comment]
@rbg-joe - I've squashed this into one commit, keeping your authoriship, and merged it into issues/commercial/709-greenwich-cobrand - thanks very much!

**Comment by rbg-joe** [comment]
Lovely thanks

**Pull request closed without merging**